From: Len Brown <len.brown@intel.com>
Subject: intel_idle: additional Haswell CPU-id
References: bnc#811928
Patch-Mainline: In lenb subtree (see below)
Git-commit: 0b15841b0bbbde75134a777c543dc51fe6187108
Git-repo: git.kernel.org/pub/scm/linux/kernel/git/lenb/linux.git

Signed-off-by: Thomas Renninger <trenn@suse.de>

There is an additional HSW CPU-id, 0x46,
which has C-states exactly like CPU-id 0x45.

Signed-off-by: Len Brown <len.brown@intel.com>

Index: linux-3.0-SLE11-SP3/drivers/idle/intel_idle.c
===================================================================
--- linux-3.0-SLE11-SP3.orig/drivers/idle/intel_idle.c
+++ linux-3.0-SLE11-SP3/drivers/idle/intel_idle.c
@@ -439,6 +439,7 @@ static int intel_idle_probe(void)
 	case 0x3C:      /* HSW */
 	case 0x3F:      /* HSW */
 	case 0x45:      /* HSW */
+	case 0x46:      /* HSW */
 		cpuidle_state_table = hsw_cstates;
 		break;
 
